assets/js/components/setup-wizard/wizard-steps.js

```diff
@@ -44,7 +44,7 @@ const STEPS = {
 		Component: WizardStepSearchConsoleProperty,
 	},
 	completeSetup: {
-		title: __( 'Finish', 'google-site-kit' ),
+		title: _x( 'Finish', 'action', 'google-site-kit' ),
```

Comment:
Is one word for the context sufficient here? "Finish" itself is a verb so I'm not sure "action" will be useful here.

Perhaps something like this?
```suggestion
		title: _x( 'Finish', 'complete module setup', 'google-site-kit' ),
```

Reply:
I think technically "action" is sufficient because a context here is only needed to indicate that in this case it's a verb and thus separate from a potentially different translation string where "finish" could be a noun. But it doesn't hurt to be more specific.

Reply:
Good point; I don't really think of "finish" as a noun as well, so in that regard I suppose it makes more sense. I think the extra context would still be useful in some cases if it were to affect the conjugation although I don't have a concrete example in mind.